src/TestSuite/Fixture/FixtureDataManager.php

```diff
@@ -205,7 +205,15 @@ protected function runOperation(array $fixtures, callable $operation): void
         $dbs = $this->fixtureConnections($fixtures);
         foreach ($dbs as $connection => $fixtures) {
             $db = ConnectionManager::get($connection);
+            if (
+                !method_exists($db, 'transactional')
+                || !method_exists($db, 'disableConstraints')
+            ) {
+                continue;
```

Comment:
Should this be calling `$operation($db, $fixtures)` so that the operation still happens?

Reply:
Yup, fixed.